specs/engine.spec/src/main/scripts/io/aklivity/zilla/specs/engine/streams/application/reconfigure.create.via.http/client.rpt

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2021-2022 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect "http://localhost:8080/"
+connected
+
+write http:method "GET"
+write http:header "If-None-Match"
+write http:header "Prefer" "wait=86400"
+
+read http:status "404" "Not Found"
+read closed
+write notify FIRST_ABORTED
+
+connect "http://localhost:8080/"
+connected
+
+write http:method "GET"
+write http:header "If-None-Match"
+write http:header "Prefer" "wait=86400"
+write close
+
+read http:status "200" "OK"
+read http:header "Etag" "AAAAAAA"
+read '{"name": "test","bindings":{"net0":{"type" : "test","kind": "proxy","exit": "app0"}}}'
```

Comment:
If you want to make this more readable, you can format it over multiple lines such as this.
```
read '{'
       '"name": "test".'
       '"bindings":`
       '{'
         '"net0":'
         '{'
           '"type": "test",'
           '"kind": "proxy",'
           '"exit": "app0"'
         '}'
       '}'
     '}'
```

Note: these should also be `yaml` syntax now. 😉